server/core/src/https/views/constants.rs

```diff
@@ -0,0 +1,7 @@
+#[derive(PartialEq, Clone)]
```

Comment:
This should have a serde::{Serialize,Deserialize} with the container attribute `rename_all="snake_case"` to allow for usage/parsing in URLs and other places